vector/src/main/res/layout/item_timeline_event_text_message_stub.xml

```diff
@@ -7,7 +7,7 @@
     android:orientation="vertical"
     tools:viewBindingIgnore="true">
 
-    <TextView
+    <io.element.android.wysiwyg.EditorStyledTextView
```

Comment:
I understand that this will be used even if the lab flag is disabled for the rich text composer. Can it be an issue?
I mean if the app crashes because of `io.element.android.wysiwyg.EditorStyledTextView`, the users will not be able to disable the lab flag to fallback to a stable version.

Reply:
I've updated so that the text view class will depend on whether the rich text editor is enabled in Labs: [`4107887`](https://github.com/vector-im/element-android/pull/8011/commits/410788771c285ce557fcaf1ce2a8e13f2954f51a)